### PR TITLE
Add dh ocp4 spec

### PR DESCRIPTION
**Pull request by hemajv**
In instances where multiple projects are working on the same cluster spec, it is possible for one team to break the other team's applications if they forget to update the service account. It would be safer to have separate cluster objects for teams.

This will create a separate cluster spec for the dh ocp4 projects

cc @HumairAK @anishasthana

### CODE REVIEW

File: vars/prod-vars.yaml [hunk 1]
```diff
@@ -199,6 +199,69 @@ clusters:
           613431313731633132613335383865303566
         tlsClientConfig:
           insecure: true
+  - name: cluster-dh-apps.ocp.prod.psi.redhat.com
+    data:
+      name: apps.ocp.prod.psi.redhat.com
```
Comment (HumairAK) [inline vars/prod-vars.yaml:204]: Can we change this name to have a dh prefix?

Comment (hemajv) [reply vars/prod-vars.yaml]: added

**Comment by HumairAK** [comment]
lgtm

**Pull request merged**